rebasehelper/helpers/koji_helper.py

```diff
@@ -307,12 +308,30 @@ def download_build(cls, session, build_id, destination):
         packages = session.listRPMs(buildID=build_id)
         rpms = []
         logs = []
+        os.makedirs(os.path.join(destination, 'RPM'), exist_ok=True)
+        os.makedirs(os.path.join(destination, 'SRPM'), exist_ok=True)
         for pkg in packages:
-            # FIXME: multiple arches
-            if pkg['arch'] not in ['noarch', 'x86_64']:
+            if pkg['arch'] not in arches:
                 continue
+            if pkg['arch'] == 'src':
```

Comment:
Why the duplicate code? You can simply skip downloading logs for `'src'` arch, can't you? RPM/SRPM should already be part of `destination`.